pom.xml

```diff
@@ -130,46 +97,23 @@
         </executions>
       </plugin>
 
-
       <plugin>
         <groupId>com.mycila</groupId>
         <artifactId>license-maven-plugin</artifactId>
-        <configuration>
-          <header>etc/license.txt</header>
+        <configuration combine.self="override">
+          <header>${rootdir}/etc/license.txt</header>
           <strictCheck>true</strictCheck>
-          <aggregate>true</aggregate>
           <excludes>
+            <exclude>README.md</exclude>
             <exclude>jitpack.yml</exclude>
             <exclude>LICENSE.txt</exclude>
             <exclude>install-jdk.sh</exclude>
+            <exclude>mvnw</exclude>
+            <exclude>mvnw.cmd</exclude>
+            <exclude>.mvn/wrapper/maven-wrapper.properties</exclude>
+            <exclude>.mvn/wrapper/MavenWrapperDownloader.java</exclude>
           </excludes>
         </configuration>
-        <executions>
-          <execution>
-            <goals>
-              <goal>check</goal>
```

Comment:
No check any longer by default? Or inherited now?

Reply:
The parent applies it